lib/ansible/modules/database/mysql/mysql_variables.py

```diff
@@ -32,6 +32,19 @@
     description:
     - If set, then sets variable value to this
     type: str
+  mode:
+    description:
+    - C(global) assigns a value to a global system variable which will be changed at runtime
+      but won't persist across server restarts.
+    - C(persist) persists a global system variable to
+      the mysqld-auto.cnf option file in the data directory (the variable will survive service restarts).
```

Comment:
Maybe also mention here that the variable will be set. It follows implicitly from the other descriptions, but being a bit more explicit never hurts in docs :)

Reply:
sounds good!